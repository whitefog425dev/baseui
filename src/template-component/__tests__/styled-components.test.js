/*
Copyright (c) 2018 Uber Technologies, Inc.

This source code is licensed under the MIT license found in the
LICENSE file in the root directory of this source tree.
*/
// @flow
import React from 'react';
import {shallow} from 'enzyme';
import {StyledRoot} from '../index';

describe('Component styled components', () => {
  test('StyledRoot - basic render', () => {
    const component = shallow(<StyledRoot $prop={false} />);

    expect(component.instance().getStyles()).toMatchSnapshot(
      'StyledRoot has correct styles when prop is set to false',
    );
  });
});
