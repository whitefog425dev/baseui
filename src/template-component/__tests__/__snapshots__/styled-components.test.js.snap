// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component styled components StyledRoot - basic render: StyledRoot has correct styles when prop is set to false 1`] = `
Object {
  "color": "$theme.colors.positive400",
  "cursor": "pointer",
}
`;
